ipaserver/plugins/session.py

```diff
@@ -23,6 +23,6 @@ def execute(self, *args, **options):
         else:
             delattr(context, 'ccache_name')
 
-        setattr(context, 'logout_cookie', '')
+        setattr(context, 'logout_cookie', 'MagBearerToken=')
```

Comment:
MagBearerToken is the value that matter sin the cookie generated by mod_auth_gssapi. Since the privsep partches removed our session code and use mod_auth_gssapi + mod_session instead, then that's what we need to use.

This patch is a fix for 6685, the part about blacklist is just a suggestion for an aditional improvement but it is not required to fix the issue.